reference/api/search.mdx

```diff
@@ -20,7 +20,7 @@ Search for documents matching a specific query in the given index.
 This is the preferred endpoint to perform search when an API key is required, as it allows for [preflight requests](https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request) to be cached. Caching preflight requests **considerably improves search speed**.
 
 <Capsule intent="note">
-By default, [this endpoint returns a maximum of 1000 results](/learn/advanced/known_limitations#maximum-number-of-results-per-search). If you want to scrape your database, use the [get documents endpoint](/reference/api/documents#get-documents) instead.
+By default, [this endpoint returns a maximum of 1000 results](/learn/advanced/known_limitations#maximum-number-of-results-per-search). If you want to scrape your database, use the [get documents endpoint](/reference/api/documents#get-documents-with-post) instead.
```

Comment:
Oh, I didn't know this was documented, but it's wrong!
Since the get/fetch documents routes are exposed for the administrator of the instance, there is no limitation on it.
You can query as many documents as you want.

Reply:
Yes, it applies to the search routes